.github/workflows/maven.yaml

```diff
@@ -188,11 +184,17 @@ jobs:
 
           diff=$((right_bound - left_bound))
           sliced_array=("${CLASSNAMES[@]:$left_bound:$diff}")
-          TEST_ARG=$(echo $sliced_array | sed 's/ /,/g')
+          TEST_ARG=$(echo ${sliced_array[@]} | sed 's/ /,/g')
```

Comment:
I had a bug here, I was only running the first test from a `sliced_array`, fixed.